src/rpm/headers/header.rs

```diff
@@ -249,7 +249,7 @@ where
             })
     }
 
-    pub(crate) fn get_entry_data_as_string_array(&self, tag: T) -> Result<&[String], RPMError> {
+    pub fn get_entry_data_as_string_array(&self, tag: T) -> Result<&[String], RPMError> {
```

Comment:
Do we really want these low level methods to be exposed?

Reply:
I think we probably do.  I mean there's just no way that we can expose helper functions for every single possible tag that someone somewhere might care about.